Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:9.11.1-slim
+
+EXPOSE 3000
+
+COPY package*.json ./
+RUN npm install
+
+COPY . .
```

Comment:
:art: where is `.` in this case? isnt it host `pwd`?

Reply:
AFAIK it is relative to the Dockerfile, but I am not entirely sure